S222309556 - Admin roster view, Activity Task List, Upload Care Plan,Caretaker create new task

s222309556 opened the pull request:
Created a Nurse Roster UI in the admin page. Created a Nurse Roster button. When clicked it will redirect to Nurse roster page.

Trello card: https://trello.com/c/LmJS4gzh/74-admin-roster-view
Test video: https://deakin.au.panopto.com/Panopto/Pages/Viewer.aspx?id=97c5733f-8802-4d3b-ad54-b14e00901d1c

s222309556 committed:
Merge pull request #5 from Gopher-Industries/master

updates

s222309556 committed:
Achini Uththara - created a Nurse Roster UI in the admin page. Created a Nurse Roster button. When clicked it will redirect to Nurse roster page.

Trello card: https://trello.com/c/LmJS4gzh/74-admin-roster-view
Test video: https://deakin.au.panopto.com/Panopto/Pages/Viewer.aspx?id=97c5733f-8802-4d3b-ad54-b14e00901d1c

s222309556 commented:
I had to align the admin interface (homepage4admin) to add the Nurse Roster icon. Please review my code, and I will resolve the conflict.

el-veee commented:
@s222309556 Not sure why I was requested to review this. I was on Guardian last trimester but not anymore

s222309556 committed:
activity changes

Commit:
Added side menu on activity task list

Trello card : https://trello.com/c/QYm4K9U7/178-add-side-menu-on-activity-task-list

Commit:
Added side menu on activity task list

Trello card : https://trello.com/c/QYm4K9U7/178-add-side-menu-on-activity-task-list

Commit:
Added side menu on activity task list

Trello card : https://trello.com/c/QYm4K9U7/178-add-side-menu-on-activity-task-list

Commit:
Caretaker is able to upload a careplan for their patients.

Trello card : https://trello.com/c/ThTltNLj/34-upload-care-plan

Test Video: https://deakin.au.panopto.com/Panopto/Pages/Viewer.aspx?id=ed0c4cf7-7b12-4649-b18d-b16f00d361e7

Commit:
Caretaker create a new task for their patients. Created an interface that allows caretaker to add new tasks.
Added a plus icon to add new task.

Trello Card: https://trello.com/c/1ryyG2Rl/87-caretaker-create-new-task

s222309556 committed:
Merge branch 'master' into s222309556
